vite.config.js

```diff
@@ -178,10 +178,11 @@ module.exports = ({ command, mode }) => {
       },
       outDir: path.resolve(__dirname, 'heroku/static'),
     },
+    worker: {
+      format: 'es'
+    },
     plugins: [
-      isProduction
```

Comment:
I think we may want to restore this

Reply:
The "production" setting here only seems to apply when running within vite in production mode, that is with "npm run serve".  This is convenient when we want to locally test the production build.

It doesn't affect our deployment at all, which is done with static files.